### DISCUSSION TITLE
当前登录存在安全风险，请使用常用设备或通过手机号登录

### DISCUSSION THREAD

**Discussion by AngelLuoLi** [DreamVoid/MiraiMC#210] [问答 / Q&A]
[10:43:12 WARN]: [MiraiMC] 登录机器人时出现异常，原因: Error(bot=Bot(3253144522), code=237, title=安全提醒, message=当前登录存在安全风险，请使用常用设备或通过手机号登录。, errorInfo=)

请问如何解决

**Comment by DreamVoid** [comment]
删掉机器人的配置文件，目录在/plugins/MiraiMC/MiraiBot/bots/

**Reply by AngelLuoLi** [reply]
删除后重启再次登入 还是会显示登入存在安全风险

**Reply by DreamVoid** [reply]
> 删除后重启再次登入 还是会显示登入存在安全风险

那就在你自己的电脑登录好机器人，然后把配置复制到服务器，你的服务器IP可能被腾讯风控了

**Reply by AngelLuoLi** [reply]
我尝试了换账号 ，本地，和其他服务器  都是显示有风险   有其他解决方法吗

**Reply by DreamVoid** [reply]
> 我尝试了换账号 ，本地，和其他服务器 都是显示有风险 有其他解决方法吗

那就正常的手机QQ登录几天，只能说你的号被风控了
